.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: complete-checking
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run linting
+        run: |
+          pip install -U pip
+          pip install norminette
+          norminette sources/ includes/
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Compile the program
+        run: make
```

Comment:
Maybe we need to point exact folder to run make?

Reply:
Yes, the current design is using a subfolder to carry the entire project...

Reply:
it might be a good idea for automation to remove the subfolder

Reply:
I can change it, what do you think?

Reply:
It's already done in the MR